autometa/common/external/samtools.py

```diff
@@ -80,6 +80,7 @@ def main(args):
 if __name__ == '__main__':
     import argparse
     import logging as logger
+    import multiprocessing as mp
```

Comment:
I wanted to make the `argparse` block standalone so that we can easily copy it to another file and run it. 
If I remove ` import multiprocessing as mp` and copy the `arparse` block and try to run it it gives me an error.

Reply:
You could `import multiprocessing as mp` in the written temporary file without having to move this in the source code